packages/module-manager/lib/runModuleManager.ts

```diff
@@ -36,12 +39,22 @@ const runModuleManager = async () => {
     )
     .replace(/ {2}"version": "[\s\S]*?",/, '  version: packageJson.version,');
 
-  fs.writeFileSync(manifestPath, updatedManifest);
-  execSync('eslint --fix ' + manifestPath, { stdio: 'inherit' });
-  await new Promise(resolve => {
-    setTimeout(resolve, 1500);
+  writeFileSync(manifestPath, updatedManifest);
+  execSync('pnpm i', {
+    stdio: 'inherit',
+    cwd: resolve('..', '..'),
+  });
+
+  execSync('pnpm -F chrome-extension lint:fix', {
+    stdio: 'inherit',
+    cwd: resolve('..', '..'),
+  });
+
+  await sleep(1000);
+  execSync('git add .', {
+    stdio: 'inherit',
+    cwd: resolve('..', '..'),
```

Comment:
Running git commands feels like it goes beyond the scope of the boilerplate. If `git add .` isn't used, we wouldn't need that hacky sleep code mentioned above either.

Reply:
Why i've decided to move it here?
To ensure users won't forget to add it to tracking, we have of course new info in README, but i've thought that's good to do it automatically.

But i'll remove it